recipes/botan/all/conanfile.py

```diff
@@ -43,6 +43,16 @@ def configure(self):
 
         if self.options.single_amalgamation:
             self.options.amalgamation = True
+            if Version(self.version) >= "2.14.0":
```

Comment:
If it's no longer available, raise an ConanInvalidConfiguration. We can't remove such option, otherwise we can break the users.